packages/map-template/src/components/Directions/Directions.jsx

```diff
@@ -232,6 +238,20 @@ function Directions({ isOpen, onBack, onSetSize, snapPointSwiped, onRouteFinishe
         setSize(snapPoints.FIT);
     }
 
+    /**
+     * Sets minZoom for a specific map provider.
+     * 
+     * @param {string} mapType 
+     * @param {number} zoomLevel 
+     */
+    function setMinZoom(mapType, zoomLevel) {
```

Comment:
Is the mapType parameter needed? As far as I can tell, there's already a mapType recoil value in the component.